src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+import java.util.Scanner;
+
 public class Duke {
+
+     // Maximum number of tasks that can be held
+    private static final int MAX_TASK = 100;
+
+    // List of all tasks
+    private static Task[] Tasks = new Task[MAX_TASK];
```

Comment:
Perhaps using "taskList" or "tasks" would represent this better? If you use "taskList", I would think that its purpose will be clear enough such that the comment on line 8 becomes unnecessary.